chart/compass/templates/tests/director/director-test.yaml

```diff
@@ -319,6 +319,1286 @@ spec:
       {{ end }}
       restartPolicy: Never
 ---
+apiVersion: "testing.kyma-project.io/v1alpha1"
+kind: TestDefinition
```

Comment:
Splitting that into multiple files would be easier to use.

Not to slow down the merge of that PR we can handle that in the next improvement that will also re-iterate over the environment variables and slim it to the only needed for the particular sub-set of tests.